base/Makefile

```diff
@@ -187,11 +187,11 @@ endef
 
 # the following excludes: libuv.a, libutf8proc.a
 
-ifneq ($(USE_SYSTEM_LIBM),0)
-$(eval $(call symlink_system_library,LIBM,$(LIBMNAME)))
-else ifneq ($(USE_SYSTEM_OPENLIBM),0)
-$(eval $(call symlink_system_library,OPENLIBM,$(LIBMNAME)))
-endif
+#ifneq ($(USE_SYSTEM_LIBM),0)
+#$(eval $(call symlink_system_library,LIBM,$(LIBMNAME)))
```

Comment:
Why did you comment these out?  You're still planning on linking against the system libm, right?  This is what will put `libm` into your `usr/lib` folder and fix the error you mentioned.

Reply:
That doesn't work on the mac...

Reply:
I think llvm probably pulls in the right libm anyways, so hopefully my latest patch (thanks to @vtjnash) helps make this PR pass.